[AI Audit] AI Audit update docs (#23046)
cloudflare/cloudflare-docs: src/content/changelog/ai-audit/2025-06-17-refresh.mdx

---
title: AI Audit refresh
description: More intuitive AI crawler management
date: 2025-06-17T11:00:00Z
---

> AI Audit dashboard has been redesigned to provide you with more intuitive and granular control over AI crawlers.

Reviewer: ```suggestion
We redesigned the AI Audit dashboard to provide more intuitive and granular control over AI crawlers.
```
